SPECS/kernel/CVE-2023-2430.nopatch

```diff
@@ -0,0 +1,4 @@
+CVE-2023-2430 - Introducing commit not present
+Introducing patch not present. Note that 5.15 doesn't even have the vulnerable file https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/io_uring/msg_ring.c?h=v5.15.137
+upstream fix: e12d7a46f65ae4b7d58a5e0c1cbfa825cf8
+But io_uring.c isn't introduced until 36404b09aa609e00f8f0108356830c22b99b3cbf; not backported to 5.15.*
```

Comment:
Nit: I think you mean msg_ring.c isn't introduced until

Reply:
absolutely, thanks!